### PR TITLE
Update preStop hooks

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Use builtin sleep hook instead of running sleep binary.

### CODE REVIEW

File: cluster/manifests/skipper/skipper-redis.yaml [hunk 1]
```diff
@@ -75,7 +75,8 @@ spec:
         lifecycle:
           preStop:
             exec:
-              command: ["sleep", "30"]
+              sleep:
```
Comment (mikkeloscar) [inline cluster/manifests/skipper/skipper-redis.yaml]: The `exec` should be replaced by `sleep`

Thanks for following up on this!

Comment (AlexanderYastrebov) [reply cluster/manifests/skipper/skipper-redis.yaml]: Late night PR :facepalm: 
Fixed!

**Comment by mikkeloscar** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**